make_sysd_rules.py

```diff
@@ -158,6 +158,15 @@ def parse_syscalls_list(path):
     "sysdeps/unix/sysv/linux/sync_file_range.c",
     "sysdeps/unix/time.c",
     "sysvipc/semtimedop.c",
+    "sysdeps/nacl/dup.c",
+    "sysdeps/nacl/dup2.c",
+    "sysdeps/nacl/dup3.c",
+    "sysdeps/nacl/fcntl.c",
+    "sysdeps/nacl/fork.c",
+    "sysdeps/nacl/pipe.c",
+    "sysdeps/nacl/pipe2.c",
+    "sysdeps/nacl/wait.c",
+    "sysdeps/nacl/waitpid.c",
     ]
```

Comment:
So make_sysd_rules.py maintains the list of NaCl GlibC system call interface we are using. Whenever we are creating our own system call interface, such as waitpid.c, we need to make sure that "sysdeps/nacl/xxx.c"(where we put that source code) is added, and any previous listing of that "xxx.c", such as "posix/waitpid.c" should be removed. One suggestion is to remove all the unused and commented out lines from this file, such as #  "posix/waitpid.c", to avoid any confusion.

Reply:
Ahh, good idea